src/main/java/Task.java

```diff
@@ -0,0 +1,29 @@
+public class Task {
+    public enum Status {
+        PENDING,
+        DONE;
+
+        public String statusToSymbol() {
+            if (this == DONE) {
+                return "[\u2713] ";
+            } else {
+                return "[\u2718] ";
+            }
+        }
+
+    }
+    String name;
+    Status status;
```

Comment:
Would it be a better practice to make these fields private? Could it be placed before the methods?